src/ServiceControl.Management.PowerShell/Cmdlets/AuditInstances/InvokeAuditUpgrade.cs

```diff
@@ -12,12 +12,6 @@ public class InvokeServiceControlAuditInstanceUpgrade : PSCmdlet
         [Parameter(Mandatory = false, HelpMessage = "Do not automatically create new queues")]
         public SwitchParameter SkipQueueCreation { get; set; }
 
-        [Parameter(Mandatory = false, HelpMessage = "Disable full text search on audit messages.")]
-        public SwitchParameter DisableFullTextSearchOnBodies { get; set; }
-
-        [Parameter(Mandatory = false, HelpMessage = "Acknowledge mandatory requirements have been met.")]
-        public string[] Acknowledgements { get; set; }
```

Comment:
This wasn't OK to remove. The checks for RavenDB broker version being 3.10 or above are dependent upon this, plus this provided a check-agnostic way to provide further acknowledgements in the future, rather than have a dozen different -ForceSomething options.

Opened https://github.com/Particular/ServiceControl/pull/3858 to restore.